spaceinvaders.py

```diff
@@ -1,648 +1,687 @@
+#!/usr/bin/env python
+
 # Space Invaders
 # Created by Lee Robinson
 
-#!/usr/bin/env python
 from pygame import *
 import sys
-from random import shuffle, randrange, choice
+from os.path import abspath, dirname
+from random import randrange, choice
+
+BASE_PATH = abspath(dirname(__file__))
+FONT_PATH = BASE_PATH + "/fonts/"
+IMAGE_PATH = BASE_PATH + "/images/"
+SOUND_PATH = BASE_PATH + "/sounds/"
 
 #           R    G    B
```

Comment:
This comment should be fixed or removed.

Reply:
'Fix double ship-image load n colors comment' (0b0ad18)